src/main/scala/learnshapeless/TaggedTypes.scala

```diff
@@ -15,7 +15,7 @@ import shapeless.test.illTyped
   * Note: the file is runnable, so you can drop in println statements to look at the values of expressions
   *
   * */
-object TaggedTypes {
+object TaggedTypes extends App {
```

Comment:
If there's anything that would really tempt me to reach out and use shapeless, this is it.

The problem is obvious to anyone that's been doing programming long enough - how do I know if this `String` I have is a `name`, `id`, `address`, or `hobby`.

In fact this is so compelling, and this is what I got most out of the session, I'd suggest to start with this exercise so the attendees will get this even if the time runs out.